tcf_website/urls.py

```diff
@@ -44,4 +44,8 @@
     path('login/collect_extra_info/', views.collect_extra_info),
     path('accounts/login/', views.login),
     path('logout/', views.logout, name='logout'),
+    path(
+        '.well-known/microsoft-identity-association.json',
```

Comment:
Is the period supposed to be there? It's not in your screenshot.

Reply:
yeah that screenshot was before, I realized after the period should be there. Lemme update the ss